src/catalog/src/remote/manager.rs

```diff
@@ -173,33 +173,38 @@ impl RemoteCatalogManager {
     }
 
     /// Fetch catalogs/schemas/tables from remote catalog manager along with max table id allocated.
-    async fn initiate_catalogs(&self) -> Result<(HashMap<String, CatalogProviderRef>, TableId)> {
+    async fn initiate_catalogs(&self) -> Result<HashMap<String, CatalogProviderRef>> {
         let mut res = HashMap::new();
-        let max_table_id = MIN_USER_TABLE_ID - 1;
 
         let mut catalogs = self.iter_remote_catalogs().await;
         while let Some(r) = catalogs.next().await {
+            let mut max_table_id = MIN_USER_TABLE_ID - 1;
```

Comment:
nit-picking: we can define another constant like `MAX_SYS_TABLE_ID` in `src/common/catalog/src/consts.rs`

Reply:
good idea.